### PR TITLE
Use Frey for setting up infrastructure and deploy the latest tusd onto it

### PR DESCRIPTION
**Pull request by kvz**
After this has been approved and merged, I would suggest moving all this to an `infra` folder inside the `tusd` repo. Then I can set up the tusd Travis to automatically deploy master builds, similar to how uppy server is now deployed (https://github.com/transloadit/uppy-server/blob/master/.travis.yml#L21). What do you think @Acconut?

### CODE REVIEW

File: infra/Freyfile.toml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+[infra.resource.aws_instance.infra-tusd-server]
+  ami = "${lookup(var.ami, var.region)}"
+  instance_type = "c3.large"
+  key_name = "infra-tusd"
+  security_groups = ["fw-infra-tusd-main"]
+  [infra.resource.aws_instance.infra-tusd-server.connection]
+    key_file = "{{{config.global.ssh.privatekey_file}}}"
+    user = "{{{config.global.ssh.user}}}"
+  [infra.resource.aws_instance.infra-tusd-server.tags]
+    Name = "${var.FREY_DOMAIN}"
```
Comment (Acconut) [inline infra/Freyfile.toml]: Is this uppercase on purpose or just by accident? Are tags usually capitalized?

File: infra/Freyfile.toml [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+      "build-essential",
+      "curl",
+      "git-core",
+      "htop",
+      "iotop",
+      "libpcre3",
+      "logtail",
+      "mlocate",
+      "mtr",
+      "mysql-client",
```
Comment (Acconut) [inline infra/Freyfile.toml]: `mysql-client` could be removed. Not sure about the others :)

**Comment by Acconut** [comment]
Just two minor things but else this LGTM. Please proceed afterwards as discussed in Slack. Thank you :heart:

File: infra/Freyfile.toml (hunk 1 above)
Comment (kvz) [inline infra/Freyfile.toml]: They are capitalized yes

File: infra/Freyfile.toml (hunk 2 above)
Comment (kvz) [inline infra/Freyfile.toml]: Will do :+1:

**Pull request merged**